src/components/build/investigationBuildPage/publish/PublishPage.scss

```diff
@@ -34,7 +34,7 @@
     }
 
     .spinner-background {
-      color: grey !important;
+      color: gray !important;
```

Comment:
Minor detail but grey is the correct spelling in UK. UK spellings are my preference where they do not collide with conventions such as 'color'.